handlers/admin/adminFailedEmailsPage.go

```diff
@@ -123,5 +123,5 @@ func AdminFailedEmailsPage(w http.ResponseWriter, r *http.Request) {
 		cd.PrevLink = "/admin/email/failed?" + prevVals.Encode()
 	}
 
-	handlers.TemplateHandler(w, r, "admin/emailFailedPage.gohtml", data)
+	handlers.TemplateHandler(w, r, "emailFailedPage.gohtml", data)
```

Comment:
@jules, this page must be in `admin/` as it's admin move the page rather than change this if required.